packages/tokens/src/bin/__mocks__/cunningham.ts

```diff
@@ -11,7 +11,7 @@ module.exports = {
         m: "1rem",
       },
       weights: {
-        medium: 400,
+        medium: "400",
```

Comment:
Why ?

Reply:
I edited the condition inside `resolveRefsAux` in the fixup to the make this work :)